apps/web/src/pages/ApplicantDashboardPage/components/MetadataDate.tsx

```diff
@@ -0,0 +1,152 @@
+import { useIntl } from "react-intl";
+
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { commonMessages } from "@gc-digital-talent/i18n";
+
+import { ApplicationStatus } from "~/utils/poolCandidate";
+
+export const ApplicationDate = ({
+  closingDate,
+  submittedAt,
+  finalDecisionAt,
+  status,
+}: {
+  closingDate?: string | null;
+  submittedAt?: string | null;
+  finalDecisionAt?: string | null;
+  status: ApplicationStatus;
+}) => {
```

Comment:
Nitpick but can we type these as an interface? We don't typically inline types like this

Reply:
Changed in [74f9438](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/74f94387d46c080ee8c94ee18897bdb729dcdacc)